tests/reversetest.ark

```diff
@@ -0,0 +1,11 @@
+{
+(import "Reverse.ark")
+    # --------------------------
+    #           Slice
```

Comment:
shouldn't it be `Reverse` :P  ?

I will rewrite the unit tests later on to use single files (one per type of test, will be a lot better), to use your format which is actually better than the current one

Reply:
I Made them only to commit , all the "important" tests were made in the file and deleted